Doc/library/uuid.rst

```diff
@@ -222,6 +222,24 @@ The :mod:`uuid` module defines the following functions:
 .. index:: single: uuid5
 
 
+.. function:: uuid6(node=None, clock_seq=None)
+
+   Generate a UUID from a sequence number and the current time according to
+   :rfc:`9562`.
+   This is an alternative to :func:`uuid1` to improve database locality.
+
+   When *node* is not specified, :func:`getnode` is used to obtain the hardware
+   address as a 48-bit positive integer. When a sequence number *clock_seq* is
+   not specified, a pseudo-random 14-bit positive integer is generated.
+
+   If *node* or *clock_seq* exceed their expected bit count, only their least
+   significant bits are kept.
+
+   .. versionadded:: next
+
+.. index:: single: uuid6
```

Comment:
Put this _before_ the function docs? Applies to the others too.

For example, go to 
https://cpython-previews--120650.org.readthedocs.build/en/120650/genindex-U.html and click uuid6 and it takes you to https://cpython-previews--120650.org.readthedocs.build/en/120650/library/uuid.html#index-12, showing uuid8, with uuid6 just above.

Reply:
Oh right. How about a separate PR for fixing the docs (and cleaning up the entire uuid module's docs instead)?

Reply:
Sure!